.swiftlint.yml

```diff
@@ -4,22 +4,12 @@ disabled_rules: # rule identifiers to exclude from running
   - nesting
   - operator_whitespace
   - trailing_whitespace
-  - type_body_length
-  - variable_name
+included: # paths to include during linting. `--path` is ignored if present.
+  - Sources
```

Comment:
Just curious, I read the description but why lint only the Sources folder? Might be a handy way to ignore dependencies but there aren't any here?